src/app/backend/handler/localehandler.go

```diff
@@ -94,7 +94,16 @@ func (handler *LocaleHandler) ServeHTTP(w http.ResponseWriter, r *http.Request)
 		// we want a different index.html (for the right locale) to be served when the page refreshes.
 		w.Header().Add("Cache-Control", "no-store")
 	}
+
```

Comment:
I'd change the code here and make the language cookie the most important, then the env variable and the least important should be the default browser header.

```go
// It does not matter if there was an error. In case the string is empty it will be overriden.
acceptLanguage, _ := r.Cookie("lang")

if len(acceptLanguage) == 0 {
    acceptLanguage = os.Getenv("ACCEPT_LANGUAGE")
}

if len(acceptLanguage) == 0 {
    acceptLanguage = r.Header.Get("Accept-Language")
}
```